coalib/results/SourceRange.py

```diff
@@ -126,6 +126,51 @@ def __json__(self, use_relpath=False):
             _dict['file'] = relpath(_dict['file'])
         return _dict
 
+    def __str__(self):
+        """
+        Creates a string representation of the SourceRange object
+        Here are doctest examples of the three different representations
+
+        >>> from os.path import abspath
+
+        >>> case1 = abspath('test_file')
+        >>> example_str1 = str(SourceRange.from_values('test_file', None, None,
+        ... None, None))
+        >>> case1 == example_str1
+        True
+
+        If the whole file is affected, then just the filename is shown.
+
+        >>> case2 = abspath('test_file') + ': L1 : L2'
+        >>> example_str2 = str(SourceRange.from_values('test_file', 1,
+        ... None, 2, None))
+        >>> case2 == example_str2
+        True
```

Comment:
Hmm I still get this error when I run the tests for the above comment of mine without the space after `...`
```
ValueError: line 14 of the docstring for coalib.results.SourceRange.SourceRange.__str__ lacks blank after ...: '...test_file'
```